<GLText> Disable depth testing when scale invariance is enabled (#384)
cruise-automation/webviz: docs/src/4.15.GLText.mdx

## Scale Invariance

`<GLText />` also supports rendering text at a constant scale regardless of the zoom level by using the optional `scaleInvariantFontSize?: number` property. If set (default is `undefined`), the text will be rendered always at the same size, resembling the behavior of the `<Text />` command.

Please note that `scaleInvariantFontSize` only works if the `billboard` property is set to true.

<GLTextScaleInvariant />

### Rendering text on top of other objects

> If scale invariance is enabled, you should make sure `<GLText />` is the last command being rendered in your world representation. That will make all text markers to be drawn on top of other visible objects.

Reviewer: ```suggestion
If scale invariance is enabled, you should make sure `<GLText />` is the last command being rendered in your world representation. That will draw these text markers on top of other visible objects.
```
